app/controllers/characters_controller.rb

```diff
@@ -320,7 +320,7 @@ def build_editor
     @groups = user.character_groups.order('name asc') + [new_group]
     use_javascript('characters/editor')
     gon.character_id = @character.try(:id) || ''
-    if @character && @character.template.nil? && @character.user == current_user
+    if @character && @character.template.nil? && @character.user == current_user # rubocop:disable Style/SafeNavigation
```

Comment:
Can we make this:

```ruby
if !@character.nil? && @character.template.nil? && @character.user == current_user
```

This should remove the need for inline-disabling the SafeNavigation thing.